easybuild/easyconfigs/v/vcflib/vcflib-1.0.9-foss-2022a-R-4.2.1.eb

```diff
@@ -0,0 +1,78 @@
+# Author: Jasper Grimm (UoY)
+# Updated: Denis Kristak (INUITS)
+# Updated: Petr Král (INUITS)
+easyblock = 'CMakeMake'
+
+name = 'vcflib'
+version = '1.0.9'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://github.com/vcflib/vcflib'
+description = """vcflib provides methods to manipulate and interpret sequence variation as it can be
+ described by VCF. The Variant Call Format (VCF) is a flat-file, tab-delimited textual format intended
+ to concisely describe reference-indexed genetic variations between individuals."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'openmp': True}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = [
+    '%(name)s-%(version)s_dep-fix.patch',
+]
+checksums = [
+    {'v1.0.9.tar.gz': 'd17fcf8a34d65f1dfecf4b4290d058be744422b6baf34ecdef8ea912d59a4569'},
+    {'vcflib-1.0.9_dep-fix.patch': '76bce1c9e40dab2ad94b455bb4487dfa24eeafcf50639db0852f5ca7a31331b6'},
+]
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+    ('pkgconf', '1.8.0'),
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('Perl', '5.34.1'),
+    ('R', '4.2.1'),
+    ('XZ', '5.2.5'),
+    ('zlib', '1.2.12'),
+    ('bzip2', '1.0.8'),
+    ('HTSlib', '1.15.1'),
+    ('tabixpp', '1.1.2'),
+    ('intervaltree', '0.1'),
+    ('fastahack', '1.0.0'),
+    ('filevercmp', '20191210'),
+    ('fsom', '20151117'),
+    ('multichoose', '1.0.3'),
+    ('smithwaterman', '20160702'),
+    ('WFA2', '2.3.3'),
+]
+
+preconfigopts = "find %(builddir)s/%(name)s-%(version)s/src -type f "
+preconfigopts += r"-regextype egrep -regex '.*\.(h|cpp)' -exec sed -i"
+preconfigopts += " -e 's|SmithWatermanGotoh.h|smithwaterman/SmithWatermanGotoh.h|g'"
+preconfigopts += " -e 's|IntervalTree.h|intervaltree/IntervalTree.h|g'"
+preconfigopts += " -e 's|multichoose.h|multichoose/multichoose.h|g' -e 's|filevercmp.h|filevercmp/filevercmp.h|g'"
+preconfigopts += " -e 's|tabix.hpp|tabixpp/tabix.hpp|g' -e 's|Fasta.h|fastahack/Fasta.h|g'"
+preconfigopts += r" -e 's|disorder.h|smithwaterman/disorder.h|g' {} \; && "
+
+configopts = "-DZIG=OFF -DWFA_GITMODULE=OFF -DWFA_INCLUDE_DIRS=$EBROOTWFA2/include/wfa2lib"
+
+postinstallcmds = ["cp -r %(builddir)s/%(name)s-%(version)s/scripts %(installdir)s"]
+
+modextrapaths = {'PATH': ['scripts']}
+
+sanity_check_paths = {
+    'files': [
+        'bin/vcffilter',
+        'bin/vcfcombine',
+        'lib/libvcflib.a',
+        'lib/pyvcflib.cpython-310-x86_64-linux-gnu.so',
```

Comment:
I wouldn't check for this file specifically due to the messy filename (which is CPU-type specific, won't work on Arm), and check whether `import pyvcflib` works instead (probably needs adding `lib` to `$PYTHONPATH` via `modextrapaths`?).

Also, no need to spread this across multiple lines then?